chart/values.yaml

```diff
@@ -0,0 +1,3071 @@
+  ## @param accountsWww.livenessProbe.failureThreshold Failure threshold for livenessProbe
+  ## @param accountsWww.livenessProbe.successThreshold Success threshold for livenessProbe
+  ##
+  livenessProbe:
+    enabled: true
+    initialDelaySeconds: 10
+    periodSeconds: 30
+    timeoutSeconds: 5
+    successThreshold: 1
+    failureThreshold: 5
+  ## @param accountsWww.readinessProbe.enabled Enable readinessProbe on accounts-www containers
+  ## @param accountsWww.readinessProbe.initialDelaySeconds Initial delay seconds for readinessProbe
+  ## @param accountsWww.readinessProbe.periodSeconds Period seconds for readinessProbe
+  ## @param accountsWww.readinessProbe.timeoutSeconds Timeout seconds for readinessProbe
+  ## @param accountsWww.readinessProbe.failureThreshold Failure threshold for readinessProbe
+  ## @param accountsWww.readinessProbe.successThreshold Success threshold for readinessProbe
+  ##
+  readinessProbe:
+    enabled: true
+    initialDelaySeconds: 10
+    periodSeconds: 30
+    timeoutSeconds: 5
+    successThreshold: 1
+    failureThreshold: 5
+  ## @param accountsWww.startupProbe.enabled Enable startupProbe on accounts-www containers
+  ## @param accountsWww.startupProbe.initialDelaySeconds Initial delay seconds for startupProbe
+  ## @param accountsWww.startupProbe.periodSeconds Period seconds for startupProbe
+  ## @param accountsWww.startupProbe.timeoutSeconds Timeout seconds for startupProbe
+  ## @param accountsWww.startupProbe.failureThreshold Failure threshold for startupProbe
+  ## @param accountsWww.startupProbe.successThreshold Success threshold for startupProbe
+  ##
+  startupProbe:
+    enabled: false
+    initialDelaySeconds: 10
+    periodSeconds: 30
+    timeoutSeconds: 5
+    successThreshold: 1
+    failureThreshold: 5
+  ## @param accountsWww.customLivenessProbe Custom livenessProbe that overrides the default one
+  ##
+  customLivenessProbe: {}
+  ## @param accountsWww.customReadinessProbe Custom readinessProbe that overrides the default one
+  ##
+  customReadinessProbe: {}
+  ## @param accountsWww.customStartupProbe Custom startupProbe that overrides the default one
+  ##
+  customStartupProbe: {}
+  ## accounts-www resource requests and limits
+  ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+  ## @param accountsWww.resources.limits The resources limits for the accounts-www containers
+  ## @param accountsWww.resources.requests The requested resources for the accounts-www containers
+  ##
+  resources:
+    limits: {}
+    requests: {}
+  ## Configure Pods Security Context
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod
+  ## @param accountsWww.podSecurityContext.enabled Enabled accounts-www pods' Security Context
+  ## @param accountsWww.podSecurityContext.fsGroup Set accounts-www pod's Security Context fsGroup
+  ##
+  podSecurityContext:
+    enabled: true
+    fsGroup: 1001
```

Comment:
Does it make sense to use a non-root gid for the filesystem if the container is running as root?

Reply:
Changed